packages/components/src/DateTimePickers/DatePicker/DatePicker.component.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import CalendarHeader from './CalendarHeader';
+import CalendarBody from './CalendarBody';
+import theme from './DatePicker.scss';
+
+function DatePicker(props) {
+	return (
+		<div className={theme.container}>
+			<CalendarHeader />
+			<hr className={theme.separator} />
+			<CalendarBody />
```

Comment:
I'm wondering if we should not put CalendarHeader and CalendarBody code here too. 
Each component is very simple, and it would make the style sync for each row (header and body rows) easier as they will be one same classname, hosted in DatePicker

Reply:
&#10003;